### PR TITLE
add Input component

### PR DESCRIPTION
**Pull request by jujuzitroni**
part of #11

heroku:
https://plants-need-input-usdnbkpjt8rl.herokuapp.com/storybook/?path=/story/component-input--input-field

### CODE REVIEW

File: src/app/components/Input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './Input.module.css';
+
+function Input(): JSX.Element {
+  return (
+    <label className={styles.container}>
+      Enter new habit
+      <input className={styles.input} placeholder="E.g. take a break" />
+    </label>
```
Comment (Dev-dfm) [inline src/app/components/Input/Input.tsx:9]: Component looks very nice, but is there a reason why you don't declare types?

**Review by Dev-dfm — APPROVED** [review]
lgtm 👍

File: src/app/components/Input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './Input.module.css';
+
+function Input(): JSX.Element {
```
Comment (lmachens) [inline src/app/components/Input/Input.tsx:4]: This component has a very generic name. An input could be a password input, a text field etc. 
But this is a specific input with a fix label and placeholder.
I would rename this to e.g. `HabitInput` or add properties for the label and placeholder, to support different kind of inputs.
In addition, you will need a `value` and `onChange` to control the input. This could be done later too.

File: src/app/globals.css [hunk 3]
```diff
@@ -18,4 +18,15 @@
...
   --deleteButton-active: #cc8a93;
-}
\ No newline at end of file
+}
+
+body {
+  margin: 0;
+  font-family: var(--main-font);
+  color: var(--main-font-color);
+}
```
Comment (lmachens) [inline src/app/globals.css:27]: Makes sense, but seems unrelated to "Add Input Component"

**Review by lmachens — COMMENTED** [review]
looks good

**Review by dhnazarenko — COMMENTED** [review]
LGTM 🚀

**Review by lajanzen — APPROVED** [review]
🚀
